### ISSUE TITLE
Accessibility: ICellRendererAngularComp and role="status" does not work quite well with Jaws 2022

### ISSUE THREAD

**Issue by filipaNobrega** [ceolter/ag-grid#5083]
In my code, there is a cell which is constantly changing its value from time to time (using SignalR). Let's consider the following code:

```html
<div role="status"><span>{{params.data.status}}</span></div>
```
```ts
@Component({
  selector: "app-status-renderer",
  templateUrl: "./template-renderer.component.html",
  styleUrls: ["./template-renderer.component.scss"],
})
export class StatusRendererComponent implements ICellRendererAngularComp {
  params: ICellRendererParams;

  agInit(params: ICellRendererParams) {
    this.params = params;
  }

  refresh(params: ICellRendererParams): boolean {
    this.params = params;
    return true;
  }
}
```

When using JAWS 2022, this is the screen reader output:
```txt
<voice name="PCCursorVoice"> Actions </voice> <voice name="PCCursorVoice"> b61d758e-9ed8-4286-80a5-d2d8e59f227e </voice> Starting
<voice name="PCCursorVoice"> Actions </voice> <voice name="PCCursorVoice"> b61d758e-9ed8-4286-80a5-d2d8e59f227e </voice> Stopping
```

When the expected should be:
```txt
Starting
Stopping
```

If instead of using `cellRendererFramework`, I use `cellRenderer`, then it works as expected.
